vendor/code-of-conduct.bn.md

```diff
@@ -1,67 +1,63 @@
+গোষ্ঠীর অবাধ এবং স্বাগতপূর্ণ ভাবমূর্তি বজায় রাখার আগ্রহে,
+আমরা কনট্রিবিউটর এবং মেনটেইনার হিসেবে অঙ্গীকার করছি যে,
+এই প্রজেক্ট ও গোষ্ঠীতে যোগদান বয়স, শরীর, অক্ষমতা, জাতি,
+লিঙ্গ পরিচয় ও অভিব্যক্তি, অভিজ্ঞতা, জাতীয়তা, চেহারা,
+বর্ণ, ধর্ম, যৌন পরিচয় ও অভিব্যক্তি নির্বিশেষে সবার জন্য
 একটি হয়রানি-মুক্ত অভিজ্ঞতা করব।
 
 
 ## আমাদের মানদন্ড
- 
+
 ইতিবাচক পরিবেশ তৈরী করতে ব্যবহারের কিছু উদাহরন:
 
 * স্বাগতপূর্ণ ভাষা ব্যবহার
 * বিভিন্ন দৃষ্টিভঙ্গি এবং অভিজ্ঞতার প্রতি সম্মান
 * মার্জিত ভাবে গঠনমূলক সমালোচনা গ্রহণ
 * গোষ্ঠীর উন্নতিকে প্রাধান্য দেওয়া
 * গোষ্ঠীর বাকি সদস্যদের প্রতি সহমর্মিতা দেখানো
- 
-  
+
+
 গ্রহণযোগ্য নয়, অংশগ্রহণকারীদের এমন আচরণের উদাহরণ:
 
 * যৌনতাপূর্ণ ভাষা বা চিত্রাবলী এবং অবাঞ্ছিত যৌন আসক্তি বা অগ্রগতি
 * ট্রলিং, অপমানজনক / অসম্মানজনক মন্তব্য এবং ব্যক্তিগত বা রাজনৈতিক আক্রমণ
 * প্রকাশ্য বা ব্যক্তিগত হয়রানি
 * অন্যদের ব্যক্তিগত তথ্য প্রকাশ, যেমন: স্পষ্ট অনুমতি ছাড়া প্রকৃত বা ইলেকট্রনিক ঠিকানা প্রকাশ
 * অন্যান্য আচরণ যা পেশাদার পরিবেশে যুক্তিসঙ্গতভাবে অনুপযুক্ত হিসেবে বিবেচনা করা হয়
-  
+
 
 ## আমাদের দায়িত্ব
 
-প্রজেক্ট মেনটেইনাররা গ্রহণযোগ্য আচরণের মান ব্যাখ্যা করা এবং অনুপযুক্ত আচরণের জন্য যথোপযুক্ত 
+প্রজেক্ট মেনটেইনাররা গ্রহণযোগ্য আচরণের মান ব্যাখ্যা করা এবং অনুপযুক্ত আচরণের জন্য যথোপযুক্ত
 এবং যথোপযোত সংশোধনমূলক ব্যবস্থা নেয়ার জন্য দায়বদ্ধ।
 
-যদি কোনও কমেন্ট, কমিট, কোড, উইকি, ইস্যু এবং বাকি অবদানগুলো আচরণ বিধি মেনে না চলে, 
-প্রজেক্ট মেনটেইনারদের তা অপসারণ, সম্পাদনা বা প্রত্যাখ্যান করতে পারে এবং যদি কারো ব্যবহার 
+যদি কোনও কমেন্ট, কমিট, কোড, উইকি, ইস্যু এবং বাকি অবদানগুলো আচরণ বিধি মেনে না চলে,
+প্রজেক্ট মেনটেইনারদের তা অপসারণ, সম্পাদনা বা প্রত্যাখ্যান করতে পারে এবং যদি কারো ব্যবহার
 অনুপযুক্ত, ভীতিপ্রদর্শক, আক্রমণাত্মক বা ক্ষতিকর মনে হলে অবদানকারীকে অস্থায়ী বা স্থায়ীভাবে নিষিদ্ধ করা হতে পারে।
 
 
 ## ব্যাপ্তি
 
-কোনও ব্যক্তি, যে প্রজেক্ট বা গোষ্ঠীর প্রতিনিধিত্ব করছে, তার জন্য এই আচরনবিধি প্রজেক্ট ও প্রকাশ্যে, উভয় ক্ষেত্রেই প্রযোজ্য।  
-প্রজেক্ট বা গোষ্ঠীর প্রতিনিধিত্ব করার উদাহরন: অফিসিয়াল ইমেইল ব্য়বহার করা, অফিসিয়াল সোশ্যাল মিডিয়া একাউন্ট ব্যবহার করা, 
-কোনো অনলাইন বা অফলাইন অনুষ্ঠানে নিযুক্ত প্রতিনিধি হিসেবে কাজ করা। প্রজেক্ট মেনটেইনাররা প্রতিনিধিত্ব-এর 
+কোনও ব্যক্তি, যে প্রজেক্ট বা গোষ্ঠীর প্রতিনিধিত্ব করছে, তার জন্য এই আচরনবিধি প্রজেক্ট ও প্রকাশ্যে, উভয় ক্ষেত্রেই প্রযোজ্য।
+প্রজেক্ট বা গোষ্ঠীর প্রতিনিধিত্ব করার উদাহরন: অফিসিয়াল ইমেইল ব্য়বহার করা, অফিসিয়াল সোশ্যাল মিডিয়া একাউন্ট ব্যবহার করা,
+কোনো অনলাইন বা অফলাইন অনুষ্ঠানে নিযুক্ত প্রতিনিধি হিসেবে কাজ করা। প্রজেক্ট মেনটেইনাররা প্রতিনিধিত্ব-এর
 সংজ্ঞা আরও ভাল ভাবে ব্যাখ্যা এবং স্পষ্ট করতে পারে।
 
 
 ## প্রয়োগ
 
-অপমানজনক, হয়রানী বা গ্রহণযোগ্য নয় এমন আচরণের খবর প্রজেক্ট টীমকে জানানো যাবে [এই ই-মেল এ]। 
-সব অভিযোগ পর্যালোচনা করা ও তদন্ত করা হবে এবং এই পরিস্থিতিতে যে প্রতিক্রিয়া প্রয়োজনীয় ও উপযুক্ত 
-বলে মনে করা হবে তা প্রয়োগ করা হবে। প্রজেক্ট টীম এই ঘটনার রিপোর্টার সম্পর্কে গোপনীয়তা বজায় রাখতে বাধ্য। 
+অপমানজনক, হয়রানী বা গ্রহণযোগ্য নয় এমন আচরণের খবর প্রজেক্ট টীমকে জানানো যাবে [INSERT EMAIL ADDRESS]।
```

Comment:
Bug from my original implementation in #13, these placeholders wouldn't be swapped out.